examples/http/web_service_performance.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * this driver monitor the performance of web services by measuring their response time.
+ * Creates a Custom Variables for each URL with its response time value:
+ *  - Label: Taken from the list of urls
+ *  - Value: The response time for each urls
+ */
+
+function sha256(message) {
```

Comment:
Why do you need to use crypto hash with sh256 encryption to generate a variable uid ?